x/clp/types/querier.go

```diff
@@ -17,15 +13,3 @@ const (
 func NewQueryReqGetPool(symbol string) PoolReq {
 	return PoolReq{Symbol: symbol}
 }
-
```

Comment:
Why are these being removed?

Reply:
Synced up with Tim offline, so we are good with remove these unused methods.